Fortran90/data_types/meso_approx.f90

```diff
@@ -142,6 +142,52 @@ subroutine approx_initialisation(appr)
      end do
      do i=2,7
       appr%hamilt%corr%term(i-1,1)=i
+      appr%hamilt%corr%intmap(i-1)=1
+     end do
+     appr%eqn%lhs(1,1)=1
+     appr%eqn%rhs(1,1)=2
+    end if
+
+    if(appr%approx.eq.'bpe') then
+     nsites=7
+     npar=1
+     allocate(appr%hamilt%orig%term(2**nsites,nsites))
+     allocate(appr%hamilt%orig%value(2**nsites))
+     allocate(appr%hamilt%corr%term(2**nsites,nsites))
+     allocate(appr%hamilt%corr%intmap(2**nsites))
+     allocate(appr%hamilt%corr%value(npar))
+     allocate(appr%eqn%lhs(npar,nsites))
+     allocate(appr%eqn%rhs(npar,nsites))
+     appr%hamilt%orig%term=0
+     appr%hamilt%orig%value=0.d0
+     appr%hamilt%corr%term=0
+     appr%hamilt%corr%intmap=0
+     appr%hamilt%corr%value=0.1d0
```

Comment:
Make a note in the code (or add an issue on Github), that this is a guess value and might need to be adjusted in the future.